Sources/SwiftGenKit/Parsers/Resources/ResourceParser.swift

```diff
@@ -0,0 +1,40 @@
+//
+// SwiftGenKit
+// Copyright © 2019 SwiftGen
+// MIT Licence
+//
+
+import Foundation
+import PathKit
+
+public enum Resource {
```

Comment:
I think that the term `Resource` can be misleading. Is it a parser to process `.rsrc` files (like back in the ResEdit days 😅 ), or some kind of other file format? I got confused at first 😉 

Naming is hard… but I suggest to use something like `FileList` instead (for the name of the enum, parser, the command, etc), what do you think?

Reply:
Naming is definitely hard 😆 
I like `FileList`, or maybe even just `Files`